ReqIFSharp/AttributeValue/AttributeValueDate.cs

```diff
@@ -64,7 +64,7 @@ internal AttributeValueDate(SpecElementWithAttributes specElAt)
         /// <summary>
         /// Gets or sets the attribute value.
         /// </summary>
-        public DateTime TheValue { get; set; }
+        public DateTime? TheValue { get; set; }
```

Comment:
same comment as above, i don't think we should make this nullable, please revert.

Reply:
Okey